string name
string model_path
string label_path
---
